src/components/layertree/LayerTree.js

```diff
@@ -242,7 +275,7 @@ class LayerTree extends PureComponent {
   /**
    * Apply a mutation to all the children recursively.
    */
-  applyToChildren(tree, item, mutation, itemIgnored) {
+  applyToChildren(tree, item, mutation, isIgnored) {
```

Comment:
Call it `isIgnoredFunc` , otherwise we have the feeling that the parameter is a boolean.